tests/rgw/utils.py

```diff
@@ -0,0 +1,22 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def clone_the_repo(config, node, path_to_clone):
```

Comment:
This would make it `rgw` specific and prevent others from leveraging the same.

IMO, we could have this method in the `utility/utils.py`. This could be done by passing the `git_url`

Reply:
makes sense. I have pushed the changes.

Reply:
I have pushed the changes.